ui/src/components/CardInput.js

```diff
@@ -65,7 +81,7 @@ class CardInput extends Component {
                     <form onSubmit={this.onSubmit}>
                             <input
                                 type="text"
-                                name="card"
+                                name="count"
```

Comment:
I think this might have been changed by mistake - we'll want to make sure the name of the text input is `card` so that the input values are updated to the `card` state field and sent in the POST request to add a new card here: https://github.com/TifMoe/go-fishbowl/blob/071ea63f655115b43a7e9014a829fc5e13346bc3/ui/src/components/CardInput.js#L44

Reply:
Ooooh yeah I think this was definitely an accident. I tried to add a secondary input for the "give me some cards" button to specify the number of cards, but then I deleted it for a later commit. It looks like part of it refused to die

Reply:
Haha, I've been there!! I thought it was probably that bc you mentioned wanting to make the number of cards an input eventually :)